web3.js/src/vote-account.ts

```diff
@@ -57,50 +100,51 @@ const VoteAccountLayout = BufferLayout.struct([
     BufferLayout.offset(BufferLayout.u32(), -8),
     'epochCredits',
   ),
+  BufferLayout.struct(
+    [BufferLayout.nu64('slot'), BufferLayout.nu64('timestamp')],
+    'lastTimestamp',
+  ),
 ]);
 
 type VoteAccountArgs = {
   nodePubkey: PublicKey;
-  authorizedVoterPubkey: PublicKey;
-  authorizedWithdrawerPubkey: PublicKey;
+  authorizedWithdrawer: PublicKey;
   commission: number;
-  votes: Array<Lockout>;
   rootSlot: number | null;
-  epoch: number;
-  credits: number;
-  lastEpochCredits: number;
-  epochCredits: Array<EpochCredits>;
+  votes: Lockout[];
+  authorizedVoters: AuthorizedVoter[];
+  priorVoters: PriorVoters;
+  epochCredits: EpochCredit[];
+  lastTimestamp: Timestamp;
 };
 
 /**
  * VoteAccount class
  */
 export class VoteAccount {
   nodePubkey: PublicKey;
-  authorizedVoterPubkey: PublicKey;
-  authorizedWithdrawerPubkey: PublicKey;
+  authorizedWithdrawer: PublicKey;
   commission: number;
-  votes: Array<Lockout>;
   rootSlot: number | null;
-  epoch: number;
-  credits: number;
-  lastEpochCredits: number;
-  epochCredits: Array<EpochCredits>;
+  votes: Lockout[];
+  authorizedVoters: AuthorizedVoter[];
+  priorVoters: PriorVoters;
+  epochCredits: EpochCredit[];
+  lastTimestamp: Timestamp;
 
   /**
    * @internal
    */
   constructor(args: VoteAccountArgs) {
     this.nodePubkey = args.nodePubkey;
-    this.authorizedVoterPubkey = args.authorizedVoterPubkey;
-    this.authorizedWithdrawerPubkey = args.authorizedWithdrawerPubkey;
+    this.authorizedWithdrawer = args.authorizedWithdrawer;
     this.commission = args.commission;
-    this.votes = args.votes;
     this.rootSlot = args.rootSlot;
-    this.epoch = args.epoch;
-    this.credits = args.credits;
-    this.lastEpochCredits = args.lastEpochCredits;
+    this.votes = args.votes;
+    this.authorizedVoters = args.authorizedVoters;
+    this.priorVoters = args.priorVoters;
```

Comment:
I suppose we could check if the `priorVoters.isEmpty` is true then send an empty buffer, rather than an array filled with `0`s